ui/src/seminars/components/SeminarTimezone.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment-timezone';
+import { Alert } from 'antd';
+
+function SeminarTimezone({ timezone, displayAsAlert }) {
+  const displayText = `Times in ${timezone} (${moment
+    .tz(timezone)
+    .format('z')})`;
+  if (displayAsAlert) {
+    return <Alert type="error" message={displayText} className="di" />;
+  }
+  return displayText;
+}
+
+SeminarTimezone.propTypes = {
+  timezone: PropTypes.string.isRequired,
+  displayAsAlert: PropTypes.bool,
```

Comment:
at the end it's in just one place you need to wrap it with a error alert, so instead of having a flag prop you can just:

`<Alert type="error" message={<SeminarTimezone timezone[timezone}/>} />` where you need it.